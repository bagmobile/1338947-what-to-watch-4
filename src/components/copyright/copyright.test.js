import React from "react";
import renderer from "react-test-renderer";
import Copyright from "./copyright";


describe(`Copyright component`, () => {

  it(`Render`, () => {
    const tree = renderer
      .create(<Copyright/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
