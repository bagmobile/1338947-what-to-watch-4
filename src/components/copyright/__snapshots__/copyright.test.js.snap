// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component Render 1`] = `
<div
  className="copyright"
>
  <p>
    Â© 2019 What to watch Ltd.
  </p>
</div>
`;
